src/helpers/rateLimit.ts

```diff
@@ -23,5 +40,11 @@ export default rateLimit({
       'too many requests, Refer: https://twitter.com/SnapshotLabs/status/1605567222713196544',
       429
     );
-  }
+  },
+  store: client
+    ? new RedisStore({
+        sendCommand: (...args: string[]) => client.sendCommand(args),
+        prefix: 'hub:'
```

Comment:
We should call it same as the repo
```suggestion
        prefix: 'snapshot-hub:'
```